mail_activity_partner/models/mail_activity.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, models, fields
+
+
+class MailActivity(models.Model):
+    _inherit = 'mail.activity'
+
+    partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        index=True,
+        compute='_compute_res_partner_id',
+        store=True,
+    )
+
+    commercial_partner_id = fields.Many2one(
+        related='partner_id.commercial_partner_id',
+        string='Customer Company',
```

Comment:
Change description to 'Commercial Entity'

Reply:
@jbeficent Changed :+1: